### PR TITLE
add config.configFile documentation

### PR DESCRIPTION
**Pull request by zethis**
Hey there,

Some documentation for multi domain compatible configuration OAuth2-proxy.
Related to issue #101 
Feel free to review it.

Have a good day everyone.

### CODE REVIEW

**Comment by pierluigilenoci** [comment]
@zethis I know it seems pointless but even a change of documentation involves a bump of the chart. 
Can you add it?

**Comment by pierluigilenoci** [comment]
@zethis I just checked the PR just now. 
You cannot add these parameters to the table that way because they aren't Chart options that can be used with a `--set` command.

So the example of how to configure is fine, adding parameters that way is not.

File: helm/oauth2-proxy/README.md [hunk 1]
```diff
@@ -100,6 +100,12 @@ Parameter | Description | Default
 `config.cookieSecret` | server specific cookie for the secret; create a new one with `openssl rand -base64 32 \| head -c 32 \| base64` | `""`
 `config.existingSecret` | existing Kubernetes secret to use for OAuth2 credentials. See [secret template](https://github.com/oauth2-proxy/manifests/blob/master/helm/oauth2-proxy/templates/secret.yaml) for the required values | `nil`
 `config.configFile` | custom [oauth2_proxy.cfg](https://github.com/oauth2-proxy/oauth2-proxy/blob/master/contrib/oauth2-proxy.cfg.example) contents for settings not overridable via environment nor command line | `""`
+`config.configFile.cookie_secure` | custom [oauth2_proxy.cfg](https://github.com/oauth2-proxy/oauth2-proxy/blob/master/contrib/oauth2-proxy.cfg.example) contents for settings cookie_secure, "true" or "false" | `""`
```
Comment (pierluigilenoci) [inline helm/oauth2-proxy/README.md:103]: They cannot be added here.

Comment (pierluigilenoci) [reply helm/oauth2-proxy/README.md:103]: Maybe a separate section on how to configure the software to use multi whitelist-domain and multi hosts would be the perfect solution.

**Comment by pierluigilenoci** [comment]
@sdomdm the chart version bump is needed

**Comment by pierluigilenoci** [comment]
@zethis could you please take a look?

**Comment by pierluigilenoci** [comment]
@desaintmartin could you please take a look?

**Comment by zethis** [comment]
LGTM 👍

**Comment by zethis** [comment]
need me to bump the version of the chart ? regarding a5157050c879f892f0815ac3b3a7a04353ca1097 commit ?

**Comment by pierluigilenoci** [comment]
@zethis I did the version bump on PR.

**Review by pierluigilenoci — APPROVED** [review]

**Pull request merged**